src/pages/dashboard/menu.js

```diff
@@ -37,36 +40,61 @@ let StyledFloatingMenu = styled(FloatingMenu)`
 const MenuPage = () => {
     const [menuId, setMenuId] = useState(null)
     const [menuData, setMenuData] = useState()
+    const [hasMenu, setHasMenu] = useState(true)
+    const [menuSelectorData, setMenuSelectorData] = useState([])
```

Comment:
If I understood correctly, this is redundant. In MenuSelector this.state.data can just be initialized as []?

Reply:
Leaving it as it is for now because we have plans to elevate the menu states to MenuPage